.changeset/on-load-plugins.md

```diff
@@ -0,0 +1,12 @@
+---
+"jspsych": minor
```

Comment:
Should this be major because it breaks on_load for custom plugins that return promises already?

Reply:
I guess that was a sort of unofficial thing that folks could have chosen to do. I don't think it matters much since the package is getting a major version bump anyways, so whatever you feel is correct is fine with me.

Reply:
Ok, I'd like to have it listed in the major section then ;)
```suggestion
"jspsych": major
```